finesse/hardware/plugins/temperature/dp9800.py

```diff
@@ -70,8 +70,8 @@ def parse_data(data: bytes) -> tuple[list[Decimal], str]:
     except UnicodeDecodeError as e:
         raise DP9800Error(e)
 
-    vals_begin = 3  # Following STX T SP
-    vals_end = 74  # Following STX T (SP %4.2f)*9, assuming 9 vals
+    vals_begin = 2  # Following STX T
+    vals_end = 74  # Following STX T (%8.2f)*9, assuming 9 vals
```

Comment:
So is there no space after the `STX` and `T` after all then? Surely if we were one char out the numbers we'd be seeing would be all wrong though?

Reply:
It's not that there isn't a space, it's that for temperatures lower than 10000, the first character will be a space because of the fixed width of each field.

Reply:
Sorry, let me rephrase that: there isn't an explicit space character. It's only if the temperature is less than 10000, in which case the fixed width of the temperature field will insert spaces.

Reply:
Ohhhhh I see. That makes sense!

Reply:
Yeah, sorry, basically I think I just took %4.2f to mean "4 characters before the decimal point, 2 after it", whereas it should be %8.2f which gives a total width of 8 characters, with 5 before the decimal point. So I guess I stuck in the `SP` to make the 5th character.